src/h256.rs

```diff
@@ -1,112 +1,79 @@
 use core::cmp::Ordering;
+use numext_fixed_hash;
 
-/// Represent 256 bits
-#[derive(Eq, PartialEq, Debug, Default, Hash, Clone, Copy)]
-pub struct H256([u8; 32]);
+pub(crate) type H256 = numext_fixed_hash::H256;
```

Comment:
We will use a wrapper anyway (`H256Ord`) why just:
```rust
pub(crate) H256(numext_fixed_hash::H256);
```
And then implement `AsRef<H256>`/`AsMut<H256>` for this structure to expose all method function from `numext_fixed_hash::H256`.